paper/paper.md

```diff
@@ -26,69 +26,66 @@ bibliography: paper.bib
+
+![Time-series plot generated by plot_contacts module of the raw vertical acceleration data labeled with detected gait events (initial contact/heel strike and final contact/toe off) and bout classifications. The start and end times of classified bouts are labeled by green and red vertical lines respectively. During the period shown, the participant walked for about 30 seconds, paused, performed 5 sit-to-stand repetitions, paused again, and continued walking for about 30 seconds.](Figure3.png){width=450px}
 
 # Algorithms
 
 GaitPy includes two key algorithms for processing raw accelerometer data to derive gait features. The first algorithm is used for detecting bouts of gait from continuous accelerometer data collected under free-living conditions and the second algorithm derives temporal and spatial features of gait from pre-identified bouts of gait. Below is a brief description of the algorithms.
 
-1.	Gait Classification
+**Gait Classification**  
 In order to derive gait features from data collected under free-living conditions, it is essential to identify periods of walking activity. GaitPy includes a pre-trained random forest [@Breiman2001] binary classifier that operates on time and frequency domain features extracted from 3-second epochs of vertical acceleration data. Prior to feature extraction, raw vertical acceleration data is down-sampled to 50Hz and band-pass filtered using a 0.5-3Hz 1st order Butterworth filter. Extracted signal features include dominant frequency, the ratio of the energy associated with the dominant frequency component to the total energy, the range of amplitude, the root mean square value of the signal, and the signal entropy. Gaitpy's classify_bouts module applies this binary classifier to input data to classify each non-overlapping 3-second epoch as either gait or not-gait.
 
-2.	Gait Features
-GaitPy implements a slightly modified version of a Gaussian continuous wavelet-based method [@McCamley2012] and an inverted pendulum model [@Zijlstra2003] to extract features from data collected during bouts of gait. 
+**Gait Features**  
+GaitPy implements a slightly modified version of a Gaussian continuous wavelet-based method [@McCamley2012] and an inverted pendulum model [@Zijlstra2003] to extract features from data collected during bouts of gait.
 
-Three post-processing steps are applied to remove extraneous stride detections beyond physiological limits. 
+Three post-processing steps are applied to remove extraneous stride detections beyond physiological limits.
 Step 1: Strides longer than 2.25 seconds or shorter than 0.25 seconds are removed. [@Najafi2003]
 Step 2: Strides with stance times exceeding 70% of the maximal stride time of 2.25 seconds are removed. [@Hollman2011]
 Step 3: Strides with an initial double support that exceed 20% of the maximal stride time of 2.25 seconds are removed. [@Hollman2011]
 
-# Availability
```

Comment:
This is linked to from the left-hand margin of the paper so in unnecessary here.